controllers/tasksRequests.js

```diff
@@ -0,0 +1,95 @@
+const { INTERNAL_SERVER_ERROR, SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+const taskRequestsModel = require("../models/taskRequests");
+
+const fetchTaskRequests = async (_, res) => {
+  try {
+    const taskRequests = await taskRequestsModel.fetchTaskRequests();
+
+    return res.status(200).json({
+      message: "Task requests returned successfully",
+      taskRequests,
+    });
+  } catch (err) {
+    logger.error("Error while fetching task requests", err);
+    return res.boom.badImplementation(INTERNAL_SERVER_ERROR);
```

Comment:
What's the value of catching the err, if error message is generic here.

Reply:
the catch here is for `exception` cases, rest of the errors are handled by returning an object from the models.